conf/waivers/20-long-term

```diff
@@ -51,7 +51,9 @@
 
 # RHEL-8: https://bugzilla.redhat.com/show_bug.cgi?id=1834716
 # RHEL-9: https://bugzilla.redhat.com/show_bug.cgi?id=1999587
+# https://issues.redhat.com/browse/RHEL-45706
 /hardening/anaconda/with-gui/[^/]+/service_avahi-daemon_disabled
+/hardening/anaconda(/with-gui)?/cis[^/]*/socket_systemd-journal-remote_disabled
```

Comment:
Does it also happen on RHEL8? Because previously the waiver was RHEL9 only.

Reply:
It doesn't now, but I expect it could happen also on RHEL8 as the OAA issue is also o RHEL8.